lib/rocket_pants/controller/jsonp.rb

```diff
@@ -32,7 +32,8 @@ def jsonp(options = {})
     private
 
     def jsonp_is_possible?
-      request.get? && response.content_type == "application/json" && jsonp_parameter.present?
+      # in rails-6 content_type also includes charset
+      request.get? && response.content_type.include?("application/json") && jsonp_parameter.present?
```

Comment:
Just to confirm, there isn't a strange edge case where we could use jsonp in APIv1?

Reply:
I'm pretty sure we don't use it, but this change should be safe. 

In rails 6 the `content_type` is 
`application/json charset=utf8` -- previously it was just `application/json`